.github/workflows/ci-ts.yaml

```
@@ -0,0 +1,52 @@
+      package_manager: pnpm
+      node_version: "22"
+      enable_tests: false  # Disable tests here since they need Docker
+      enable_coverage: false
+      test_command: "test"
+      lint_command: "check"
+      format_command: "check"
+
+  test-with-docker:
+    runs-on: ubuntu-latest
+    needs: ts-checks
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)